CMakeLists.txt

```diff
@@ -293,6 +293,20 @@ if(WITH_LIBUSB)
   endif(LIBUSB_1_FOUND)
 endif(WITH_LIBUSB)
 
+# realsense2
```

Comment:
This block (lines 296 - 308) should be removed.
The option (line 297) will be created by `PCL_ADD_GRABBER_DEPENDENCY` macro.
The lines 299 - 306 (without surrounding if/endif) should go into a new file `cmake/Modules/FindRSSDK2.cmake`.